aggregate.go

```diff
@@ -98,7 +98,12 @@ type AggregateConfigurer interface {
 	//
 	// It MUST be called exactly once within a single call to Configure().
 	//
-	// Each handler within an application MUST have a unique, non-empty name.
+	// The name MUST be a non-empty UTF-8 string consisting solely of printable
+	// unicode characters. A printable character is any character from the
```

Comment:
I wonder if `unicode` should start with the capital U as a standard name.

Reply:
I agree, it does seem to be capitalised everywhere its used on the web, including in Go lang docs.

Reply:
Yep, seems to be the case.